tsconfig.json

```diff
@@ -12,7 +12,7 @@
 		"noImplicitThis": true,
 		"noImplicitAny": true,
 		"strictNullChecks": true,
-		"suppressImplicitAnyIndexErrors": true,
+		//"suppressImplicitAnyIndexErrors": true,
```

Comment:
please no commented code. Why do you want to disable this?

Reply:
It just gave me an error since it's removed in typescript 5.5, I didn't configure the ts server to use the project's ts version.

It's fine on 4.6.2, so I might as well uncomment it/not disable it

Reply:
Okay. makes sense. Then just remove the whole setting